scripts/aem.js

```diff
@@ -503,7 +503,8 @@ function decorateSections(main) {
  */
 // eslint-disable-next-line import/prefer-default-export
 async function fetchPlaceholders(prefix = 'default') {
-  const overrides = getMetadata('placeholders') || '';
+  const overrides = getMetadata('placeholders') || getMetadata('root')?.replace(/\/$/, '/placeholders.json') || '';
```

Comment:
Is this something we would want to add to the base aem-boilerplate too? You are in `aem.js` afterall :)

Reply:
Maybe? This has already been updated in our `main` including the dot notation logic.

Reply:
I'm afraid this is not the only historical change in `aem.js`. I think this is part of a general cleanup of the Boilerplate. I will look into this on a different PR>